diff --git a/node_modules/@openzeppelin/contracts/governance/IGovernor.sol b/node_modules/@openzeppelin/contracts/governance/IGovernor.sol
index 8b21d29..fd6d35f 100644
--- a/node_modules/@openzeppelin/contracts/governance/IGovernor.sol
+++ b/node_modules/@openzeppelin/contracts/governance/IGovernor.sol
@@ -26,7 +26,7 @@ abstract contract IGovernor is IERC165 {
      * @dev Emitted when a proposal is created.
      */
     event ProposalCreated(
-        uint256 proposalId,
+        uint256 indexed proposalId,
         address proposer,
         address[] targets,
         uint256[] values,
@@ -40,19 +40,19 @@ abstract contract IGovernor is IERC165 {
     /**
      * @dev Emitted when a proposal is canceled.
      */
-    event ProposalCanceled(uint256 proposalId);
+    event ProposalCanceled(uint256 indexed proposalId);
 
     /**
      * @dev Emitted when a proposal is executed.
      */
-    event ProposalExecuted(uint256 proposalId);
+    event ProposalExecuted(uint256 indexed proposalId);
 
     /**
      * @dev Emitted when a vote is cast without params.
      *
      * Note: `support` values should be seen as buckets. Their interpretation depends on the voting module used.
      */
-    event VoteCast(address indexed voter, uint256 proposalId, uint8 support, uint256 weight, string reason);
+    event VoteCast(address indexed voter, uint256 indexed proposalId, uint8 support, uint256 weight, string reason);
 
     /**
      * @dev Emitted when a vote is cast with params.
