diff --git a/node_modules/@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol b/node_modules/@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol
index eec8e9b..b32bba9 100644
--- a/node_modules/@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol
+++ b/node_modules/@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol
@@ -379,7 +379,8 @@ abstract contract GovernorUpgradeable is Initializable, ContextUpgradeable, ERC1
         address[] memory targets,
         uint256[] memory values,
         bytes[] memory calldatas,
-        bytes32 descriptionHash
+        bytes32 descriptionHash,
+        string calldata reason
     ) internal virtual returns (uint256) {
         uint256 proposalId = hashProposal(targets, values, calldatas, descriptionHash);
         ProposalState status = state(proposalId);
@@ -390,7 +391,7 @@ abstract contract GovernorUpgradeable is Initializable, ContextUpgradeable, ERC1
         );
         _proposals[proposalId].canceled = true;
 
-        emit ProposalCanceled(proposalId);
+        emit ProposalCanceled(proposalId,reason);
 
         return proposalId;
     }
diff --git a/node_modules/@openzeppelin/contracts-upgradeable/governance/IGovernorUpgradeable.sol b/node_modules/@openzeppelin/contracts-upgradeable/governance/IGovernorUpgradeable.sol
index 0f84c32..f6b10e9 100644
--- a/node_modules/@openzeppelin/contracts-upgradeable/governance/IGovernorUpgradeable.sol
+++ b/node_modules/@openzeppelin/contracts-upgradeable/governance/IGovernorUpgradeable.sol
@@ -32,7 +32,7 @@ abstract contract IGovernorUpgradeable is Initializable, IERC165Upgradeable {
      * @dev Emitted when a proposal is created.
      */
     event ProposalCreated(
-        uint256 proposalId,
+        uint256 indexed proposalId,
         address proposer,
         address[] targets,
         uint256[] values,
@@ -46,19 +46,19 @@ abstract contract IGovernorUpgradeable is Initializable, IERC165Upgradeable {
     /**
      * @dev Emitted when a proposal is canceled.
      */
-    event ProposalCanceled(uint256 proposalId);
+    event ProposalCanceled(uint256 indexed proposalId, string reason);
 
     /**
      * @dev Emitted when a proposal is executed.
      */
-    event ProposalExecuted(uint256 proposalId);
+    event ProposalExecuted(uint256 indexed proposalId);
 
     /**
      * @dev Emitted when a vote is cast without params.
      *
      * Note: `support` values should be seen as buckets. Their interpretation depends on the voting module used.
      */
-    event VoteCast(address indexed voter, uint256 proposalId, uint8 support, uint256 weight, string reason);
+    event VoteCast(address indexed voter, uint256 indexed proposalId, uint8 support, uint256 weight, string reason);
 
     /**
      * @dev Emitted when a vote is cast with params.
